src/jupyter-aou/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: "2.4"
+services:
+  app:
+    container_name: "application-server"
+    image: "us-west2-docker.pkg.dev/shared-pub-buckets-94mvrf/workbench-artifacts/app-aou-jupyter@sha256:bd995ffb80b8b1b14f122b6ceb9e66c2037bbb9b182b4501c61b944cef4220a2"
+    user: "jupyter"
+    restart: always
+    volumes:
+      - .:/workspace:cached
+    ports:
+      - "8888:8888"
+    networks:
+      - app-network
+    cap_add:
+      - SYS_ADMIN
```

Comment:
So sounds like we decided the existing resource mount setup is OK security wise?

Reply:
Sounds good.